### DISCUSSION TITLE
Ableton Live

### DISCUSSION THREAD

**Discussion by onealhtml** [catppuccin/catppuccin#2535] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

It would be nice to have my DAW themed like the rest of my PC. Ableton has a 'highlight color,' so it could be implemented like Mocha-Rosewater.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closing since https://github.com/catppuccin/catppuccin/issues/2839 has been created and is in-review.
